docs/about/history.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Valkey history"
+linkTitle: "History"
+weight: 4
+description: How the Valkey project started
+aliases:
+    - /topics/history
+---
+
+Valkey is a fork of the open-source Redis (REmote DIctionary Server) database
+created in 2009 by the Italian hacker Salvatore “antirez” Sanfilippo. He
+announced it on [Hacker News][hn] on Feb 25, 2009. [GitHub][gh] and
+[Instagram][insta] were among the early adopters.
+
+[hn]: https://news.ycombinator.com/item?id=494649
+[gh]: https://github.blog/2009-11-03-introducing-resque/
+[insta]: https://instagram-engineering.com/storing-hundreds-of-millions-of-simple-key-value-pairs-in-redis-1091ae80f74c
+
+Early works of Salvatore Sanfilippo
```

Comment:
This section seems a bit out of place, as it really has little to do with Redis or Valkey.

Reply:
I trimmed it down a bit.

Reply:
I agree it's cool all the stuff Salvatore did, it just didn't seem all that relevant to the history. I'm fine leaving it.

Reply:
There's no other place to put it. :D

Reply:
A blog post? We also have that support on the website. "An homage to Salvatore"

Reply:
I'll keep it here for now. We can move this later when we have a blog.